de.metas.marketing/base/src/main/java/de/metas/marketing/base/UserService.java

```diff
@@ -58,7 +58,13 @@ public void updateUserFromContactPersonIfFeasible(
 			return; // no user to update the email
 		}
 
-		final User user = userRepo.getById(userId);
+		final User user = userRepo.getByIdInTrx(userId);
+
+		if (user == null)
```

Comment:
this one shall not be null (see above), so the null check is pointless.

Reply:
Null check deleted.